index.js

```diff
@@ -0,0 +1,2 @@
+require('./format-as-currency');
```

Comment:
@bcherny Here are my results:

Using `umd`:

`Webpack`
```
09:24:49 webpack.1 | http://localhost:3808/webpack-dev-server/
09:24:49 webpack.1 | webpack result is served from //localhost:3808/webpack/
09:24:49 webpack.1 | content is served from /Users/user/Dev/test-project
09:24:52 webpack.1 | Hash: e90fe8b1dc1a01f282a4
09:24:52 webpack.1 | Version: webpack 1.12.13
09:24:52 webpack.1 | Time: 2547ms
09:24:52 webpack.1 |          Asset     Size  Chunks             Chunk Names
09:24:52 webpack.1 | application.js  5.93 MB       0  [emitted]  application
09:24:52 webpack.1 |  manifest.json  0 bytes          [emitted]  
09:24:52 webpack.1 | chunk    {0} application.js (application) 2.28 MB [rendered]
09:24:52 webpack.1 |     [0] ./webpack/application.js 1.47 kB {0} [built]
09:24:52 webpack.1 |     [1] ./~/angular/index.js 48 bytes {0} [built]
09:24:52 webpack.1 |     [2] ./~/angular/angular.js 1.13 MB {0} [built]
09:24:52 webpack.1 |     [3] ./~/angular-resource/index.js 62 bytes {0} [built]
09:24:52 webpack.1 |     [4] ./~/angular-resource/angular-resource.js 31.3 kB {0} [built]
09:24:52 webpack.1 |     [5] ./~/angular-animate/index.js 60 bytes {0} [built]
09:24:52 webpack.1 |     [6] ./~/angular-animate/angular-animate.js 150 kB {0} [built]
09:24:52 webpack.1 |     [7] ./~/angular-messages/index.js 62 bytes {0} [built]
09:24:52 webpack.1 |     [8] ./~/angular-messages/angular-messages.js 25.8 kB {0} [built]
09:24:52 webpack.1 |     [9] ./~/angular-aria/index.js 54 bytes {0} [built]
09:24:52 webpack.1 |    [10] ./~/angular-aria/angular-aria.js 14.9 kB {0} [built]
09:24:52 webpack.1 |    [11] ./~/format-as-currency/dist/format-as-currency.js 5.48 kB {0} [built]
09:24:52 webpack.1 |    [12] ./~/angular-ui-mask/index.js 150 bytes {0} [built]
09:24:52 webpack.1 |    [13] ./~/angular-ui-mask/dist/mask.js 40.6 kB {0} [built]
09:24:52 webpack.1 |    [14] ./~/angular-material/index.js 270 bytes {0} [built]
09:24:52 webpack.1 |    [15] ./~/angular-material/angular-material.js 849 kB {0} [built]
09:24:52 webpack.1 |    [16] ./~/angular-material-data-table/index.js 124 bytes {0} [built]
09:24:52 webpack.1 |    [17] ./~/angular-material-data-table/dist/md-data-table.js 36.9 kB {0} [built]
09:24:52 webpack.1 | 
09:24:52 webpack.1 | WARNING in ./~/format-as-currency/dist/format-as-currency.js
09:24:52 webpack.1 | Critical dependencies:
09:24:52 webpack.1 | 1:489-496 This seems to be a pre-built javascript file. Though this is possible, it's not recommended. Try to require the original source to get better results.
09:24:52 webpack.1 |  @ ./~/format-as-currency/dist/format-as-currency.js 1:489-496
09:24:52 webpack.1 | webpack: bundle is now VALID.
```

`application.js`
```js
import formatAsCurrency from "format-as-currency"
console.log(formatAsCurrency);
// Object {}
```

I cannot do the following since `formatAsCurrency` is an empty object:
```js
import angular from "angular"
import formatAsCurrency from "format-as-currency"
angular.module("myapp", [formatAsCurrency]);
```

This works:
```js
import angular from "angular"
import formatAsCurrency from "format-as-currency"
angular.module("myapp", ["formatAsCurrency"]);
```
---

Using `patch-1`

`terminal`
```
09:25:29 webpack.1 | http://localhost:3808/webpack-dev-server/
09:25:29 webpack.1 | webpack result is served from //localhost:3808/webpack/
09:25:29 webpack.1 | content is served from /Users/user/Dev/test-project
09:25:31 webpack.1 | Hash: 8dd701cd6d3ad50ee90b
09:25:31 webpack.1 | Version: webpack 1.12.13
09:25:31 webpack.1 | Time: 2582ms
09:25:31 webpack.1 |          Asset     Size  Chunks             Chunk Names
09:25:31 webpack.1 | application.js  5.93 MB       0  [emitted]  application
09:25:31 webpack.1 |  manifest.json  0 bytes          [emitted]  
09:25:31 webpack.1 | chunk    {0} application.js (application) 2.28 MB [rendered]
09:25:31 webpack.1 |     [0] ./webpack/application.js 1.47 kB {0} [built]
09:25:31 webpack.1 |     [1] ./~/angular/index.js 48 bytes {0} [built]
09:25:31 webpack.1 |     [2] ./~/angular/angular.js 1.13 MB {0} [built]
09:25:31 webpack.1 |     [3] ./~/angular-resource/index.js 62 bytes {0} [built]
09:25:31 webpack.1 |     [4] ./~/angular-resource/angular-resource.js 31.3 kB {0} [built]
09:25:31 webpack.1 |     [5] ./~/angular-animate/index.js 60 bytes {0} [built]
09:25:31 webpack.1 |     [6] ./~/angular-animate/angular-animate.js 150 kB {0} [built]
09:25:31 webpack.1 |     [7] ./~/angular-messages/index.js 62 bytes {0} [built]
09:25:31 webpack.1 |     [8] ./~/angular-messages/angular-messages.js 25.8 kB {0} [built]
09:25:31 webpack.1 |     [9] ./~/angular-aria/index.js 54 bytes {0} [built]
09:25:31 webpack.1 |    [10] ./~/angular-aria/angular-aria.js 14.9 kB {0} [built]
09:25:31 webpack.1 |    [11] ./~/format-as-currency/index.js 72 bytes {0} [built]
09:25:31 webpack.1 |    [12] ./~/format-as-currency/format-as-currency.js 4.6 kB {0} [built]
09:25:31 webpack.1 |    [13] ./~/angular-ui-mask/index.js 150 bytes {0} [built]
09:25:31 webpack.1 |    [14] ./~/angular-ui-mask/dist/mask.js 40.6 kB {0} [built]
09:25:31 webpack.1 |    [15] ./~/angular-material/index.js 270 bytes {0} [built]
09:25:31 webpack.1 |    [16] ./~/angular-material/angular-material.js 849 kB {0} [built]
09:25:31 webpack.1 |    [17] ./~/angular-material-data-table/index.js 124 bytes {0} [built]
09:25:31 webpack.1 |    [18] ./~/angular-material-data-table/dist/md-data-table.js 36.9 kB {0} [built]
09:25:31 webpack.1 | webpack: bundle is now VALID.
```

`application.js`
```js
import formatAsCurrency from "format-as-currency"
console.log(formatAsCurrency);
// "format-as-currency"
```

This works correctly since `formatAsCurrency` contains the module name:
```js
import angular from "angular"
import formatAsCurrency from "format-as-currency"
angular.module("myapp", [formatAsCurrency]);
```